src/components/02-components/card-number/card-number.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+export type BrandProps = {
+  heading?: string;
+  subHeading?: string;
+  cta: string;
+  ctaTitle: string;
+  variation: string;
+};
+
+export default function CardNumber({ heading, subHeading, cta, ctaTitle, variation }: BrandProps) {
+
+  const theme = useTheme();
+
+  // Variables according to the variant of position 
+  let position;
```

Comment:
See comment above on using constants instead of normal variables. Also, consider renaming `variant` to something more clear.